utils/swift_build_support/swift_build_support/xcrun.py

```diff
@@ -43,3 +43,19 @@ def find(tool, sdk=None, toolchain=None):
         return str(out.rstrip().decode())
     except subprocess.CalledProcessError:
         return None
+
+
+@cache_util.cached
+def sdk_path(sdk):
+    """
+    Return the path string for given SDK, according to `xcrun --show-sdk-path`.
+
+    If `xcrun --show-sdk-path` cannot find the SDK, return None.
+    """
+    command = ['xcrun', '--sdk', sdk, '--show-sdk-path']
+    try:
+        out = subprocess.check_output(command,
+                                      stderr=subprocess.PIPE)
```

Comment:
Does `stderr=subprocess.PIPE` mean that the stderr would not be printed to the terminal?  Any particular reason to do this?  I'm afraid that in case `xcrun` fails in CI (but still exits with 0 exit code) we'd be left with no clues left in the build log.